src/gifcodec.c

```diff
@@ -422,9 +422,9 @@ gdip_load_gif_image (void *stream, GpImage **image, BOOL from_file)
 		si = gif->SavedImages[i];
 		img_desc = &si.ImageDesc;
 		if (img_desc->Top < 0 || img_desc->Height < 0 ||
-		    img_desc->Left < 0 || img_desc->Width < 0 ||
-		    (img_desc->Width + img_desc->Left) > screen_width ||
-		    (img_desc->Height + img_desc->Top) > screen_height) {
+			img_desc->Left < 0 || img_desc->Width < 0 ||
+			(img_desc->Width + img_desc->Left) > screen_width ||
+			(img_desc->Height + img_desc->Top) > screen_height) {
```

Comment:
good example of where we should just revert to the previous version with tab + space for alignment. (essentially every change in this file)